terraform/terraform.tfvars.json

```diff
@@ -36,6 +36,24 @@
       "s3_vpc_endpoint_ids": ["vpce-07c8b9bc931b5d915", "vpce-0b4969d793365d5d4", "vpce-0ae35d9cfc690bc1d"],
       "default_chunk_size": "20000"
     },
+    "preqa": {
+      "name": "preqa",
+      "account_id": "492687888235",
+      "vpc_id": "vpc-037acd53d9ce813b4",
+      "is_production": false,
+      "db_subnet_prefix": "preproduction",
+      "s3_path": "live2",
+      "sirius_env": "casrecdmpq",
+      "sirius-api-role-suffix": "20210816120609472800000010",
+      "sirius-exec_role-suffix": "2021081612060893390000000a",
+      "run_api_tests": "False",
```

Comment:
Why not running API tests on the new env?

Reply:
Because we're already running on pre which is effectively same env and we can run them manually for testing purposes though I suppose i could run them